test_util/arraylist.cpp

```diff
@@ -17,21 +17,29 @@
   along with OPM.  If not, see <http://www.gnu.org/licenses/>.
 */
 
+#include <opm/io/eclipse/EclFile.hpp>
+#include <opm/io/eclipse/ERst.hpp>
+
+#include <cstdint>
+#include <cstddef>
+#include <cstdlib>
+#include <filesystem>
 #include <iomanip>
 #include <iostream>
-#include <tuple>
-#include <getopt.h>
-#include <filesystem>
 #include <sstream>
+#include <tuple>
+#include <vector>
+#include <string>
 
-#include <opm/io/eclipse/EclFile.hpp>
-#include <opm/io/eclipse/ERst.hpp>
+#include <getopt.h>
 
 using EclEntry = Opm::EclIO::EclFile::EclEntry;
 
-static void printHelp() {
+namespace {
 
-    std::cout << "Usage: arraylist [OPTIONS] ECL_FILE_NAME"<<std::endl
+static void printHelp()
```

Comment:
butter up that bacon boy (both static and anon)

Reply:
:facepalm: 

I've pushed an update to address this.